AltUseVendors/__init__.py

```diff
@@ -378,6 +404,10 @@ def PerformedSecondaryUseAction(self, caller: unrealsdk.UObject, function: unrea
 
         info.purchase_function(caller, vendor)
 
+        # Play the dialogue sound if one is given for this vendor
+        if vendor.InteractiveObjectDefinition.name in AKE_DICT[Game.GetCurrent()]:
+            vendor.PlayAkEvent(unrealsdk.FindObject("AkEvent", AKE_DICT[Game.GetCurrent()][vendor.InteractiveObjectDefinition.name]))
```

Comment:
as above, `vendor.InteractiveObjectDefinition.Name`

you can save doing this dict lookup twice using something like
```py
event = AKE_DICT[Game.GetCurrent()].get(vendor.InteractiveObjectDefinition.Name)
if event is not None:
  ...
```

Reply:
Done